examples/generate_kmers_bodo.py

```diff
@@ -0,0 +1,62 @@
+import glob
```

Comment:
Do we need a comment here to explain this example?

Reply:
I don't have enough context. @marquisdepolis could you suggest a short description for the example?

Reply:
Thanks, added.